httpd.go

```diff
@@ -61,3 +78,11 @@ func messageHandler(p Param) (int, string) {
 		return 200, fmt.Sprintf("Message sent successfully to %s", p.Channel)
 	}
 }
+
+func sendAsync(p Param, postTime time.Time, ch chan error) (int, string) {
```

Comment:
`sendLater()` or so is more appropriate because it doesn't send message asynchronous.